src/index.js

```diff
@@ -1,13 +1,16 @@
 import './style.css';
-import Tasks from './Tasks.js';
+import Tasks from './modules/Tasks.js';
+import { clearCompleted, clearTasks, updateStatus } from './modules/status.js';
 
 const MyTasks = new Tasks();
 
 const addInput = document.querySelector('.newItem');
+const clearCompletedBtn = document.querySelector('.clearBtn');
+const clearAll = document.querySelector('.fa-arrows-rotate');
 
 const editAction = () => {
   document.querySelectorAll('.task').forEach((val) => {
-    val.children[0].children[1].addEventListener('blur', (e) => {
+    val.children[1].children[0].addEventListener('blur', (e) => {
       /* eslint-disable */
```

Comment:
Please ensure that `linters` are not disabled. A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))).